src/pkg/k8s/configmap.go

```diff
@@ -29,13 +28,11 @@ func (k *K8s) CreateConfigmap(namespace, name string, data map[string][]byte) (*
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
+			Labels:    make(Labels),
```

Comment:
```suggestion
			Labels:    k.Labels,
```

These should all be `k.Labels` so that the common labels on the struct are still applied

Reply:
Using `k.Labels` in `GenerateSecret()` is resulting in labels being incorrectly applied to zarf generated secrets:

current zarf secret labels:

```
Name: private-registry Labels: {"app.kubernetes.io/managed-by":"zarf"}
Name: private-git-server Labels: {"app.kubernetes.io/managed-by":"zarf"}
Name: zarf-package-init Labels: {"app.kubernetes.io/managed-by":"zarf","package-deploy-info":"init"}
```

after using `k.Labels` in `GenerateSecret()`:

```
Name: private-registry Labels: {"app":"zarf-injector","app.kubernetes.io/managed-by":"zarf","package-deploy-info":"init","zarf.dev/agent":"ignore"}
Name: private-git-server Labels: {"app":"zarf-injector","app.kubernetes.io/managed-by":"zarf","package-deploy-info":"init","zarf.dev/agent":"ignore"}
Name: zarf-package-init Labels: {"app":"zarf-injector","app.kubernetes.io/managed-by":"zarf","package-deploy-info":"init","zarf.dev/agent":"ignore"}
```
need to take a closer look and figure out why

Reply:
As discussed synchronously, directly setting the objects `Labels` field with `k.Labels` results in undesirable side effects because `k.Labels` becomes populated with data that may or may not be relevant or accurate for use across multiple objects.